components/ReviewCard.tsx

```diff
@@ -1,18 +1,30 @@
-import React from 'react'
+import React, { useRef } from 'react'
 import Markdown from 'markdown-to-jsx'
 import gitDiffParser, { File } from 'gitdiff-parser'
 import ReactDiffViewer from 'react-diff-viewer'
 import Prism from 'prismjs'
+import { ACCEPT_SUBMISSION, REJECT_SUBMISSION } from '../graphql/queries'
+import { useMutation } from '@apollo/react-hooks'
 import _ from 'lodash'
 import { SubmissionData } from '../@types/submission'
 
 type ReviewCardProps = {
+  session: any
   submissionData: SubmissionData
 }
 
-export const ReviewCard: React.FC<ReviewCardProps> = ({ submissionData }) => {
-  const diff = _.get(submissionData, 'diff', '')
+export const ReviewCard: React.FC<ReviewCardProps> = ({
+  session,
+  submissionData
+}) => {
+  const diff = _.get(submissionData, 'diff', '').replace(
```

Comment:
@songz  and I spoke and we should remove the regex and let users fix the submissions themselves. The CLI is fixed so we'll let them reconcile the data themselves